helm/orca/config/alerts-mapping.yaml

```
@@ -185,6 +185,416 @@ prometheus:
         kind: node
         properties:
           name: instance
+    - name: KubeCronJobRunning
+      source_mapping:
+        origin: kubernetes
+        kind: cronjob
```

```suggestion
        kind: cron_job
```
Let's follow the convention of using underscores for kind names consisting of multiple parts. Note e.g. `persistent_volume_claim` of `daemon_set `.